slamon_afm/models.py

```diff
@@ -7,10 +8,24 @@
     TypeDecorator
 from sqlalchemy.orm import relationship, backref
 from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy import func
+from sqlalchemy import func, event, case, literal_column
 
 db = SQLAlchemy()
 
+from slamon_afm.stats import statsd
+
+
+class TaskException(Exception):
+    task = None
+
+    def __init__(self, *args, task=None):
```

Comment:
Is it really necessary to get the `*args` anyways and then pass it to the `Exception` class? Python tutorials do not show this kind of behavior: https://docs.python.org/3.4/tutorial/errors.html#user-defined-exceptions

According to that page, the correct way would be:

```python
class TaskException(Exception):
    def __init__(self, task):
        self.task = task
```